src/pages/library/datasetBuilder/DatasetBuilder.ts

```diff
@@ -96,6 +104,7 @@ const ObjectSetListSection = <T extends DatasetBuilderType>({
           value,
           checked: isChecked(objectSet, value),
           onChange: (value) => onChange(value, objectSet.header),
+          icon: objectSet.makeIcon ? objectSet.makeIcon(value, objectSet.header) : undefined,
```

Comment:
Can `?.` be used here?
```suggestion
          icon: objectSet.makeIcon?.(value, objectSet.header),
```

Reply:
Oh that's cool...just tested that out and I think it should work